python/lsst/ts/standardscripts/maintel/integration_tests/ccw_rotator_stress_test.py

```diff
@@ -0,0 +1,266 @@
+      in the other direction while the rotator is still slewing. Repeat this
+      test after a configurable series of time delay after slewing started.
+
+
+
+    Warning:
+    --------
+    This script requires to slew the telescope. Make sure the system is ready
+    to slew before running the script.
+
+    """
+
+    def __init__(self, index):
+        super().__init__(
+            index=index,
+            descr="Perform stress test between Camera Cable Wrap (CCW) and Rotator.",
+        )
+
+        self.mtcs = MTCS(
+            domain=self.domain, log=self.log, intended_usage=MTCSUsages.Slew
+        )
+
+        # How long to track for before each new slew (in seconds).
+        self.track_time = 15.0
+
+        # Configuration parameters.
+        self.n_rot_pos = 5
+        self.n_pos_repeat = 3
+        self.time_delays = np.array([1.0, 2.0, 5.0, 10.0])
+
+        # Ignore CSCs that are not part of the test. This will allow us to run
+        # the test if these CSCs are not enabled or being used for something
+        # else.
+        self.mtcs.check.mtaos = False
+        self.mtcs.check.mtm1m3 = False
+        self.mtcs.check.mtm2 = False
+        self.mtcs.check.hexapod_1 = False
+        self.mtcs.check.hexapod_2 = False
+        self.mtcs.check.dome = False
+        self.mtcs.check.mtdometrajectory = False
+
+    def set_metadata(self, metadata):
+        metadata.duration = 60  # rough estimate
+
+    @classmethod
+    def get_schema(cls):
+        schema = """
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: https://github.com/lsst-ts/ts_standardscripts/maintel/CcwRotatorStressTest.yaml
+        title: CcwRotatorStressTest v1
+        description: Configuration for CcwRotatorStressTest.
+        type: object
+        properties:
+            n_rot_pos:
+                description: Number of random rotator positions to select.
+                type: integer
+                default: 5
+                minimum: 0
+            n_pos_repeat:
+                description: Number of times to repeat each random position.
+                type: integer
+                default: 3
+                minimum: 1
+            time_delays:
+                description: List of delays to send new targets after slew (in seconds).
+                type: array
+                default: [1., 2., 5., 10.]
+                items:
+                    type: number
+                    minimum: 1.
+                    maximum: 30.
+        additionalProperties: false
+        """
+        return yaml.safe_load(schema)
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration.
+        """
+        self.n_rot_pos = int(config.n_rot_pos)
+        self.n_pos_repeat = int(config.n_pos_repeat)
+        self.time_delays = np.array(config.time_delays)
```

Comment:
Please consider `self.config = config` to avoid all this duplication.

Also please remove the type casting because your schema specifies the types. If your code actually needed a numpy array then that last cast would be fine, but it only needs a sequence and it is sure to get that.